mods/farming/api.lua

```diff
@@ -315,10 +315,10 @@ farming.register_plant = function(name, def)
 	for i = 1, def.steps do
 		local drop = {
 			items = {
-				{items = {mname .. ":" .. pname}, rarity = 9 - i},
-				{items = {mname .. ":" .. pname}, rarity= 18 - i * 2},
-				{items = {mname .. ":seed_" .. pname}, rarity = 9 - i},
-				{items = {mname .. ":seed_" .. pname}, rarity = 18 - i * 2},
+				{items = {mname .. ":" .. pname}, rarity = 9 - ( i / 8 ) * def.steps},
```

Comment:
Not sure about either yet, still thinking.
When i = 1 rarity should be 8
When i = def.steps rarity should be 1